release-notes/v1.3.1.yaml

```diff
@@ -0,0 +1,28 @@
+date: March 4, 2025
+
+# Changes that are expected to cause an incompatibility with previous versions, such as deletions or modifications to existing APIs.
+breaking changes: |
+  Use the envoy JSON formatter for the default access log instead of text formatter.
+
+# Updates addressing vulnerabilities, security flaws, or compliance requirements.
+security updates: |
+  Fixed CVE-2025-25294.
+
+# New features or capabilities added in this release.
+new features: |
+  Add defaulter for gateway-api resources loading from file to be able to set default values.
```

Comment:
this also fixes a bug, which support empty values in API fields which have in built defaults